### PR TITLE
bugfix/v1-070/fixed courses ui bugs

### PR DESCRIPTION
**Pull request by Julia-Maliutsina**
Fixed some bugs on cutting course description to a certain length, font sizes and course cards sizes and margins.

### CODE REVIEW

**Review by irina-sinitsyna — DISMISSED** [review]

**Review by ostapdans99 — CHANGES_REQUESTED** [review]

File: frontend/src/components/Course/Course.tsx [hunk 1]
```diff
@@ -73,10 +73,18 @@ const CourseItem: React.FC<Props> = ({
       {pageName === PAGES.myCourses && status !== COURSE_STATUSES.completed && (
         <MobileCourseProgress>
           <ProgressBar
-            value={progress}
-            text={`${progress}%`}
             size="medium"
-            variant="mobileCourse"
+            variant={
```
[the remark this answers is not in the document]
Comment (Julia-Maliutsina) [reply frontend/src/components/Course/Course.tsx:78]: fixed

File: frontend/src/components/Course/Course.tsx [hunk 2]
```diff
@@ -73,10 +73,18 @@ const CourseItem: React.FC<Props> = ({
...
+              (status &&
+                [COURSE_STATUSES.successful, COURSE_STATUSES.completed].includes(status) &&
+                VARIANTS.successfulMobile) ||
+              VARIANTS.incompleteMobile
+            }
+            text={
```
[the remark this answers is not in the document]
Comment (Julia-Maliutsina) [reply frontend/src/components/Course/Course.tsx:83]: fixed

[the remark this answers is not in the document]
Comment (k-krasilnikova) [reply frontend/src/components/Course/Course.tsx:83]: I still can't see calculating in constant..
Where you moved it?

[the remark this answers is not in the document]
Comment (Julia-Maliutsina) [reply frontend/src/components/Course/Course.tsx]: Moved to constant inside CourseContainer.

**Review by ostapdans99 — APPROVED** [review]

**Review by k-krasilnikova — APPROVED** [review]

**Pull request merged**